src/node/historical_queries.h

```diff
@@ -386,47 +342,6 @@ namespace ccf::historical
       }
     }
 
-    // Returns true if this is a valid signature that passes our verification
-    // checks
-    bool verify_signature(const StorePtr& sig_store, ccf::SeqNo sig_seqno)
```

Comment:
Correct me if I'm wrong but I think we can actually delete retired nodes from the store now, as historical queries were the only reason we couldn't complete this work before (see #2474).

Reply:
That's correct, but best done in a separate PR.